velox/core/QueryConfig.h

```diff
@@ -101,25 +101,6 @@ class QueryConfig {
   static constexpr const char* kCastMatchStructByName =
       "cast_match_struct_by_name";
 
-  /// If set, cast from float/double/decimal/string to integer truncates the
-  /// decimal part, otherwise rounds.
-  static constexpr const char* kCastToIntByTruncate = "cast_to_int_by_truncate";
```

Comment:
Is this config used only by Spark? If true, should be safe to remove it.

Reply:
I think so. Would be nice to remove. Thanks.